CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
-# I have *literally* no idea what I'm doing.
-# Know any better? Feel free to correct my (probably) awful mistakes.
-# :)
-
 cmake_minimum_required(VERSION 3.1)
 
 project(disccord)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wno-sign-append") # I added these flags, change them if you don't like them
+set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wno-sign-append")
```

Comment:
Wouldn't it be better to use something like this:

```cmake
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror -Wno-sign-append")
```

so that you're appending to the existing C++ flags? Or is this intentional?